packs/feats/come-and-get-me.json

```
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You open yourself to attacks so you can respond in turn. Until your rage ends, you are @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard], and damage rolls against you gain a +2 circumstance bonus. If a creature hits you, that creature is off-guard to you until the end of your next turn. If you hit it before the end of your next turn, you gain temporary Hit Points equal to your Constitution modifier, or double that on a critical hit. These temporary Hit Points last until the end of your rage.</p>"
+            "value": "<p>You open yourself to attacks so you can respond in turn. Until your rage ends, you are @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard], and damage rolls against you gain a +2 circumstance bonus. If a creature hits you, that creature is off-guard to you until the end of your next turn. If you hit it before the end of your next turn, you gain temporary you gain temporary @Damage[(@actor.level)[healing]]{Hit Points equal to your Constitution modifier}, or @Damage[(2*(@actor.level))[healing]]{double} that on a critical hit. These temporary Hit Points last until the end of your rage.</p>"
```

```suggestion
            "value": "<p>You open yourself to attacks so you can respond in turn. Until your rage ends, you are @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard], and damage rolls against you gain a +2 circumstance bonus. If a creature hits you, that creature is off-guard to you until the end of your next turn. If you hit it before the end of your next turn, you gain temporary you gain temporary Hit Points equal to your Constitution modifier, or double that on a critical hit. These temporary Hit Points last until the end of your rage.</p>"
```
Make an effect for the temp HP and link it in a note on the other effect.